go/vt/vtgate/planbuilder/testdata/unsupported_cases.txt

```diff
@@ -329,37 +329,37 @@ Gen4 plan same as above
 Gen4 plan same as above
 
 "select keyspace_id from user_index where id = 1 and id = 2"
-"unsupported: where clause for vindex function must be of the form id = <val> (multiple filters)"
+"unsupported: where clause for vindex function must be of the form id = <val> or id in(<val>,...) (multiple filters)"
 Gen4 plan same as above
 
 "select keyspace_id from user_index where func(id)"
-"unsupported: where clause for vindex function must be of the form id = <val> (not a comparison)"
+"unsupported: where clause for vindex function must be of the form id = <val> or id in(<val>,...) (not a comparison)"
```

Comment:
Maybe we can stop calling them unsupported, some of them do not seem reasonable queries.

Reply:
We can do it in different PR as well.

Reply:
I'm not sure what you mean here. We should call some of them invalid?